content/docs/command-reference/exp/show.md

```diff
@@ -0,0 +1,112 @@
+# exp show
+
+Print a customizable table of `dvc experiments`, their metrics and parameters.
+
+> Press `q` to exit.
+
+## Synopsis
+
+```usage
+usage: dvc exp show [-h] [-q | -v] [-a] [-T] [--all-commits] [--no-pager]
+                    [--include-metrics <metrics_list>]
+                    [--exclude-metrics <metrics_list>]
+                    [--include-params <params_list>]
+                    [--exclude-params <params_list>]
+                    [--sort-by <metric/param>] [--sort-order {asc,desc}]
+                    [--no-timestamp] [--sha] [--show-json]
+                    [--precision <n>]
+```
+
+## Description
+
+Displays experiments in a detailed table which includes their parent, name (or
+hash), metrics, and parameters. Only the experiments derived from the Git `HEAD`
+are shown by default but all experiments can be included with the
+`--all-commits` option. Example:
+
+```dvc
+$ dvc exp show
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2               │
+│ master        │  0.55259 │ 0.91536 │ 50         │ 2               │
+│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
+│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
+└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+```
+
+Your terminal will enter a
+[paginated screen](/doc/command-reference/dag#paginating-the-output) by default,
+which you can typically exit by typing `Q`. Use `--no-pager` to print the table
+to standard output.
+
+By default, the printed experiments table will include columns for all metrics
+and params from the entire project. The `--include-metrics`,
+`--exclude-metrics`, `--include-params`, and `--exclude-params` options can be
+used to choose them.
+
+Experiments in the table are first grouped (by parent commit). They are then
+sorted inside each group, chronologically by default. The `--sort-by` and
+`--sort-order` options can change this ordering, based on any single metric or
+param.
```

Comment:
Probably last Q (while we wait for approval) @pmrowla:

Can you sort by a column you're not currently showing (e.g. a param hidden with `--exclude-params`)? Thanks

Reply:
No, if you try to sort by a column that is excluded, the `dvc exp show` command will error out.

Reply:
Cool, noted in the ref.